### ISSUE TITLE
Lookup source behind the quote

### ISSUE THREAD

**Issue by mnunemacher-dm** [defmethodinc/just-not-sorry#64]
It would be nice to be able to get more information/reasoning behind the quote suggesting why a word/phrase should not be used.  Since each quote has a source url behind the scenes, maybe the block displaying the quote to the user can hyperlink to that source url.

**Comment by ProfJanetDavis** [comment]
I like this idea a lot. It could be quite motivational to read the article the quote comes from.

**Comment by sbrudz** [comment]
I agree.  This could build upon the work that @steveatdm did in the [usability styling branch](https://github.com/defmethodinc/just-not-sorry/tree/usability-styling) to change the tooltip into a real floating overlay.  @steveatdm would it be possible to modify your implementation so that it doesn't use jQuery?  Early on, just-not-sorry used jQuery, but we hit issues with our version of jQuery conflicting with the version of jQuery loaded by other extensions and causing them to break, so we removed it.

**Comment by steveatdm** [comment]
@sbrudz Yes, this was one of the next issues to tackle on the priority list. Want to remove that as a dependency and scope down the mouse move detection from the whole document to the target div (and probably some other minor refactoring).
